### ISSUE TITLE
Use ISO date for Export file names

### ISSUE THREAD

**Issue by sonora** [osmandapp/OsmAnd#16142]
Can I please suggest we use the ISO date for our .osf export file names, it is both more speaking and can be sorted more easiliy: So instead of _Export_03-01-23.osf_  use _Export_2023-01-03.osf._

**Comment by scaidermern** [comment]
This has already been requested at #14094 and #14830.

**Comment by sonora** [comment]
I see... looks like quite a few requests fell on deaf ears so far ... :wink:

**Comment by vshcherb** [comment]
Yes I think for export it could be done , though for Track I vote against it. Obviously it's not that critical cause it could be sorted by file modified date.
